src/main/webapp/calendar.html

```diff
@@ -3,14 +3,18 @@
   <head>
     <meta charset='utf-8' />
     <title>TravIS Calendar</title>
-    <link href='lib/fullcalendar/main.css' rel='stylesheet' />
+    <link href='lib/fullcalendar/main.css' rel='stylesheet'/>
```

Comment:
nit: There's no change here, and I think convention is to have a space, so I know this is really nitpicky but my nit is to change that back.